data/furniture/workspace_feeding_without_wall.urdf

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" ?>
```

Comment:
This is just a suggestion, but it bothers me how we are keeping two near-identical urdfs here. If we're keeping both with and without wall urdfs, can we have the one with wall import the other (with xacros) so that we're not keeping duplicates? Or create the collision wall programmatically in the demo?